README.md

```diff
@@ -6,13 +6,13 @@ Cloudfuse uses file caching to provide the performance of local storage, or you
 Cloudfuse is a fork of [blobfuse2](https://github.com/Azure/azure-storage-fuse), and adds S3 support, a GUI, and Windows support.
 Cloudfuse supports clouds with an S3 or Azure interface.
 
-## Installation
-Download the provided installation packages for your preferred operating system.  -- COMING SOON!
+### Windows installation
 
-### Windows
+ Download and the .exe installer from our latest release [here](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes. WinSFP is required for Cloudfuse.

```

Comment:
I don't know that you need to section on WinFSP, since the installer will prompt the user to install that.